app-web/src/assets/blog/devhub.md

```diff
@@ -104,6 +104,23 @@ There are several ways you can explore content through the Devhub. Some of the p
 - Viewing a Topic: Whenever you click on a topic you get to view the topic plus all of the individual resources
 that make up the topic. 
 
+### How to Suggest a new content for the Devhub
+
+New content can be added to the Dehub, by either suggesting a new topic or editing an existing topic.
+
+  - Suggesting a new topic: You can find the `Suggest New Topic ` button on the topics page.
+
+    Clicking the button re-directs you to a form. The form prompts the user to enter essential information regarding the topic and lets, them add
```

Comment:
there are 2 extra comma in this sentence

Reply:
Hmm, I completely missed that my bad. I'll make the necessary changes :)